lib/gpumon_config.ml

```diff
@@ -1,6 +1,14 @@
 open Stdext
 open Fun
-open Result
+open Rresult
+
+(** Like List.map, but will return an Error if and when f returns an Error. *)
+let bind_map f items =
+  let rec aux acc = function
+    | item :: rest -> (f item) >>= (fun result -> aux (result :: acc) rest)
```

Comment:
The parentheses around `(f item)` and `(fun .. rest)` are not required -- but also do no harm.